docs/server-api-auth.md

```diff
@@ -0,0 +1,145 @@
+---
+id: server-api-auth
+title: Set up server API authorization
+sidebar_label: API auth setup
+---
+
+## Overview
+
+Temporal supports the ability to restrict API call access via an optional authorization mechanism. It enables the server to process or deny individual calls based on a caller's criteria. Criteria can include the caller's unique permissions, the name of the API, and the target Namespace.
```

Comment:
The generally accepted practice I believe is to put every sentence on a new line. That makes reviewing future changes easier because only an edited sentence will be highlighted as changed.

Reply:
Hmm, interesting - I actually never encountered this before. But I suppose it makes sense if the single newlines are ignored when converting to html. I will definitely think about this for the long term - basically what are the downsides of have newlines between sentences that are otherwise meant to be paragraphs.